docs/source/model/dependencies.rst

```diff
@@ -269,7 +269,7 @@ it does **not** preserve the relative paths that they were originally residing w
 ``code/src/utils.py``. As a result, it has to be imported as ``from utils import my_func``, instead of ``from src.utils import my_func``.
 However, this may not be pleasant, as the import path is different from the original training script.
 
-To workaround this issue, the ``code_path`` should specify the parent directory, which is ``code_path=["src"]`` in this example.
+To workaround this issue, the ``code_paths`` should specify the parent directory, which is ``code_paths=["src"]`` in this example.
 This way, MLflow will copy the entire ``src/`` directory under ``code/`` and your model code will be able to import ``src.utils``.
 
 .. code-block:: python
```

Comment:
While current pyfunc.log_model only has code_path no code_paths, the below example is not correct. Should we either update to code_path or add another param for code_paths?

Reply:
I'm filing a followup PR to create the alias for pyfunc `code_paths` (add) maps to `code_path` with a deprecation warning for using `code_path` in pyfunc.